hawkrest/management/commands/hawkrequest.py

```diff
@@ -1,47 +1,69 @@
-from optparse import make_option
-
 from django.core.management.base import BaseCommand, CommandError
-
-import logging
 from mohawk import Sender
 
-from hawkrest import lookup_credentials
+from hawkrest import HawkAuthentication
+
+
+DEFAULT_HTTP_METHOD = 'GET'
+
+CMD_OPTIONS = {
+    '--url': {
+        'action': 'store',
+        'type': str,
+        'help': 'Absolute URL to request.'
+    },
+    '--creds': {
+        'action': 'store',
+        'type': str,
+        'help': 'ID for Hawk credentials.'
+    },
+    '-X': {
+        'action': 'store',
+        'type': str,
+        'help': 'Request method. Default: {}.'.format(DEFAULT_HTTP_METHOD),
+        'default': DEFAULT_HTTP_METHOD
+    },
+    '-d': {
+        'action': 'store',
+        'type': str,
+        'help': 'Query string parameters.'
+    }
+}
+
+
+def request(url, method, data, headers):
+    import requests
+    do_request = getattr(requests, method.lower())
+    res = do_request(url, data=data, headers=headers)
+    return res
+
+
+def lookup_credentials(creds_key):
+    return HawkAuthentication().hawk_credentials_lookup(creds_key)
 
 
 class Command(BaseCommand):
     help = 'Make a Hawk authenticated request'
-    option_list = BaseCommand.option_list + (
-        make_option('--url', action='store', type=str,
-                    help='Absolute URL to request.'),
-        make_option('--creds', action='store', type=str,
-                    help='ID for Hawk credentials.'),
-        make_option('-X', action='store', type=str,
-                    help='Request method. Default: %default.',
-                    default='GET'),
-        make_option('-d', action='store', type=str,
-                    help='Query string parameters'),
-    )
 
-    def handle(self, *args, **options):
-        hawk_log = logging.getLogger('mohawk')
-        hawk_log.setLevel(logging.DEBUG)
```

Comment:
It looks like you removed this. Was the logging too noisy? I had found it helpful in the past because it shows exactly what was used to create MACs, etc. If you still want to remove it, maybe there can be an option to re-enable it?